getting-started/installation.md

```diff
@@ -14,18 +14,18 @@ installers can be found under:
 
 ## Manual Installation
 
+These instructions deploy a standalone version of Magic Mirror, for [Server Only](/getting-started/installation.html#server-only) and [Client Only](/getting-started/installation.html#client-only) installs please read the install specific instructions below
+
```

Comment:
@rejas last time you lintered the lines to a max. length, will you do this here before/after merge?

Reply:
after merge I will lint again